samples/tfm_integration/tfm_psa_arch_tests/README.rst

```diff
@@ -0,0 +1,127 @@
+.. _tfm_psa_arch_api:
+
+TF-M psa-arch-tests
+###################
+
+Overview
+********
+This is a sample for Zephyr PSA level 1 with TF-M integrated.
+
+Zephyr uses Trusted Firmware (TF-M) Platform Security Architecture (PSA) APIs
+to run in a secure configuration with Zephyr itself in a non-secure configuration.
+The sample prints test info to the console as a multi-thread application.
+
+The psa-arch-tests provides differents test suits, this example supports
```

Comment:
I think you're saying you can specify one of these specific test suites. If so, I'd rewrite this as:
```
The psa-arch-tests provides different test suite configurations:
``crypto``, ``protected_storage``, or ``initial_attestation``, as set in
the sample's ``prj.conf`` file, for example:

.. code-block:: bash

   CONFIG_API_TEST_SUIT="crypto"
```

(This also fixes some misspellings)